src/redux/selectors/blockUi.js

```diff
@@ -1,5 +1,4 @@
 export const blockUi = state =>
     state.organisationUnits.loading ||
     state.reportPeriod.loading ||
-    state.reportData.loading ||
```

Comment:
I think this should def. block the UI state. Users tend to press buttons multiple times which would fire the ajax multiple times (especially if the loading spinner of the report is below the fold)

Reply:
I disagree, and will keep things this way. In most DHIS2 apps we show an inline loader for things like lists and tables. I would like to keep it that way.

Reply:
This would make the report being loaded many times if the report is loaded below the screen and the user clicks the button. Then the submit btn needs to be disabled during loading.

Reply:
Done